posthog/api/test/test_property_definition.py

```diff
@@ -32,7 +32,9 @@ def setUpTestData(cls):
         cls.user.current_team = cls.demo_team
         cls.user.save()
         EventProperty.objects.create(team=cls.demo_team, event="$pageview", property="$browser")
+        # put the same event against more than one property to test that we only count it once
         EventProperty.objects.create(team=cls.demo_team, event="$pageview", property="first_visit")
+        EventProperty.objects.create(team=cls.demo_team, event="another_event", property="first_visit")
```

Comment:
Here's the meat of what drives the fix. 

By putting the `first_visit` property on two events we see failing tests because it's double-counted